lib/chai/core/assertions.js

```diff
@@ -679,14 +684,15 @@ module.exports = function (chai, _) {
   /**
    * ### .least(value)
    *
-   * Asserts that the target is greater than or equal to `value`.
+   * Asserts that the number target is greater than or equal to `value`.
```

Comment:
Yeah, the reason it was phrased that way is because that's how it was already phrased elsewhere in the code (see: https://github.com/chaijs/chai/pull/692#issuecomment-216054505). I think the thing for us to do is submit a separate PR that rephrases all of them (both old and new) at once.